app/models/ems_refresh/save_inventory_physical_infra.rb

```diff
@@ -21,10 +23,7 @@ def save_ems_physical_infra_inventory(ems, hashes, target = nil)
       _log.debug("#{log_header} hashes:\n#{YAML.dump(hashes)}")
     end
 
-    child_keys = [
-      :physical_servers,
-      :customization_scripts
-    ]
+    child_keys = %i(physical_servers physical_racks customization_scripts)
```

Comment:
I could be wrong, but this feels backwards...I would expect the racks to be saved first, and *then* the servers, since there is a physical_rack_id on the physicals_servers that would need to be resolved.  @agrare Does that sound right?

Reply:
Actually, are there servers that aren't in racks at all?  If so, that could be where I'm getting confused.

Reply:
Yes, we have servers that aren't either in a rack nor in chassis